### ISSUE TITLE
Star+ addon default audio

### ISSUE THREAD

**Issue by Stravioska** [matthuisman/slyguy.addons#598]
in the playback setting > default audio, if i leave blank the addon play the movie in the default audio that the movie was filmed on, (even if i change the kodi language), i tried putting "por", "por-br" and "por-bra" (por-bra works on Disney+ addon) but didn't work, only plays the default of the movie, what i need to put to play the movies in pt br using the star+ addon?

**Comment by reasont** [comment]
> in the playback setting > default audio, if i leave blank the addon play the movie in the default audio that the movie was filmed on, (even if i change the kodi language), i tried putting "por", "por-br" and "por-bra" (por-bra works on Disney+ addon) but didn't work, only plays the default of the movie, what i need to put to play the movies in pt br using the star+ addon?
![121](https://github.com/matthuisman/slyguy.addons/assets/157194685/aa18db51-8789-4f87-a9e2-bc981a5b79d9)

**Comment by matthuisman** [comment]
try es-419 ?

**Comment by reasont** [comment]
> try **es-419**

Doesn't works continue playing in English, no Portuguese o Español Latinoamericano or even play in Español de España.

**Comment by matthuisman** [comment]
Are those languages even available on the content? Eg. Can you switch to it
while playing?

This setting doesn't add new audio. It simply allows automatically
selecting an audio. But that audio must already be available

**Comment by reasont** [comment]
Yes, those audios are available, it would be Latin American Spanish, I am from Costa Rica and those audios come out and I can change them manually when the video is playing, I will leave you an image. 
![Uploading 222222.png…]()

**Comment by matthuisman** [comment]
unfortunately i dont have a Star+ login to test with.

I tried with Disney+, and es-419 under "Default Audio" worked for it.


Do you maybe have a login that I can use to test and confirm the bug?
Also, what version Kodi are you using?
If you can help, my email is: blog@matthuisman.nz

**Comment by matthuisman** [comment]
this addon no longer exists so closing this issue
